src/Query/SearchQuery.php

```diff
@@ -197,7 +197,8 @@ public function toArray(): array
             'synonym_prefix' => $this->synonymPrefix,
             'text_match_type' => $this->textMatchType,
             'typo_tokens_threshold' => $this->typoTokensThreshold,
-            'vector_query' => $this->vectorQuery?->toArray(),
+            // @phpstan-ignore-next-line (->toArray is part of vectorInterface)
```

Comment:
Don't ignore anything, nothing has been ignored so far

Reply:
Issue was because you checked instanceof VectorQueryInterface on voiceQuery instead of vectorQuery